acme_diags/driver/streamflow_driver.py

```diff
@@ -80,8 +80,8 @@ def run_diag(parameter):
         using_ref_mat_file = False
         if parameter.gauges_path is None:
             raise Exception(
-                'To use a non-GSIM reference, please specify streamflow_param.gauges_path. This might be /{}/{}'.format(
-                    parameter.reference_data_path.strip('/'),
+                'To use a non-GSIM reference, please specify streamflow_param.gauges_path. This might be {}/{}'.format(
+                    parameter.reference_data_path.rstrip('/'),
```

Comment:
The `strip` to `rstrip` changes are the only changes outside the `tests` directory. Without this change, users can't specify relative paths to data.